### ISSUE TITLE
App won't launch

### ISSUE THREAD

**Issue by drickerby** [AkiniKites/AloysAdjustments#9]
I've downloaded the latest release and extracted it from the .zip, but when double clicking the .exe nothing happens. The app doesn't launch. 

I have .Net Core 3.1.11 installed, but still nothing. Any advice?

Thanks!

**Comment by AkiniKites** [comment]
Did it generate an error.log file?

**Comment by drickerby** [comment]
It did, I'm so sorry. Should have thought to include that. Duh. Output below:

`System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\******\AppData\Roaming\Aloy's Adjustments'.
   at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path, Boolean ignoreNotFound)
   at System.IO.Enumeration.FileSystemEnumerator`1.Init()
   at System.IO.Enumeration.FileSystemEnumerator`1..ctor(String directory, Boolean isNormalized, EnumerationOptions options)
   at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String directory, FindTransform transform, EnumerationOptions options, Boolean isNormalized)
   at System.IO.Enumeration.FileSystemEnumerableFactory.UserFiles(String directory, String expression, EnumerationOptions options)
   at System.IO.Directory.InternalEnumeratePaths(String path, String searchPattern, SearchTarget searchTarget, EnumerationOptions options)
   at System.IO.Directory.GetFiles(String path)
   at AloysAdjustments.Utility.FileBackup.GetMatchingFiles(String path)+MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at AloysAdjustments.Utility.FileBackup.RunWithBackup(String path, Func`1 action)
   at AloysAdjustments.Configuration.SettingsManager.Load()
   at AloysAdjustments.Main.LoadConfigs()
   at AloysAdjustments.Main..ctor()
   at AloysAdjustments.Program.Main()`

**Comment by drickerby** [comment]
OK I just manually made a folder at that path and now it opens, at least. Do you think it's a permissions thing that the app can't write to my user profile? I even tried running it as administrator just in case it couldn't get access, but I never thought to just make the path itself.

Something to look at maybe? Unless I'm just an outlier.

Cheers for the mod!

**Comment by drickerby** [comment]
Now I seem to have another issue, which is again probably only affecting a few people.

I run my PC on a 4k TV so naturally my resolution is 3840x2160. As it's nearly impossible to use Windows at this resolution on a 57" TV from a distance, I have Windows scaling set to 300%. Unfortunately, the program doesn't play nice with Windows scaling and is basically unusable at 300% scaling because none of the GUI elements are visible. See attached screenshot for what I mean: 
![aloysadjustments_scaling](https://user-images.githubusercontent.com/5812341/106522120-1d76c000-64d7-11eb-8c87-66a4211090d3.PNG)

I've tried fiddling with the high-DPI settings on the compatibility tab in properties for the .exe, but it doesn't make any difference. The only way I was able to view the program normally was to turn scaling back to 100%, and now my eyes hurt.
